### DISCUSSION TITLE
Resizing Input Image

### DISCUSSION THREAD

**Discussion by Ab-hinav** [bowenc0221/panoptic-deeplab#110] [General]
my image size is - 512 x 1024 , i am trying to train on a resize image of - (200,300). Error - Pool kernel size ?

**Comment by bowenc0221** [comment]
Error is from here: https://github.com/facebookresearch/detectron2/blob/c081505af16f54c5a013916b81636d03f7c0a8fd/detectron2/layers/aspp.py#L131-L136

**Reply by shivam228** [reply]
I am facing the same issue then how can we fix this error like how can we train it for lower resolution ? Do we need to make changes in architecture?
